Update new workflow after repo separation (#321)
near/rainbow-bridge-cli: README.md

You should observe the change of the ERC20 balance as reported by the CLI.

## Contract Development Workflow

Above steps are ways to run a local bridge and development workflows you need if make any changes to rainbow-bridge-cli. If you want to update any of solidity or rust contracts, they're not in this repo now and workflow is as following.

> - Install dependencies, doing `index.js clean`, `prepare`, `start near-node` `start ganache`

Reviewer: I suggest the following format, notice that bash commands have separate large boxes that makes them easy to copy:

- Install dependencies:
```bash
node index.js clean
node index.js prepare
```
- Start local NEAR network and Ganache
```bash
node start near-node
node start ganache
```

> - If you want to modify solidity contracts, cd to `node_modules/rainbow-bridge-sol`, doing changes there and run `../build_all.sh` to recompile solidity contracts.

Reviewer: ```suggestion
- If you want to modify solidity contracts, go to `node_modules/rainbow-bridge-sol`, make changes there and run `./build_all.sh` to recompile solidity contracts.
```

> - If you want to modify rust contracts, cd to `node_modules/ranbow-bridge-rs`, doing changes there and run `./build_all.sh` to recompile rust contracts.

Reviewer: ```suggestion
- If you want to modify rust contracts, go to `node_modules/ranbow-bridge-rs`, make changes there and run `./build_all.sh` to recompile rust contracts.
```

> - If you want to modify rainbow bridge lib,, cd to `node_modules/rainbow-bridge-lib`, doing changes there and CLI will depends on the updated `rainbow-bridge-lib`

Reviewer: ```suggestion
- If you want to modify rainbow bridge lib, go to `node_modules/rainbow-bridge-lib` and make changes there
```

- Follow instructions above to init eth contracts and near contracts, start services and start testing with bridge

> - After you're done with solidity contract updates, doing the same update and create a pull request in https://github.com/near/rainbow-bridge-sol. After you're done with rust contract updates, create PR in https://github.com/near/rainbow-bridge-rs. After you're done with rainbow bridge lib updates, create PR in https://github.com/near/rainbow-bridge-lib.

Reviewer: ```suggestion
- For changes to Solidity contract, Rust contract, and rainbow-bridge-lib, please submit PRs to: https://github.com/near/rainbow-bridge-sol , https://github.com/near/rainbow-bridge-rs , and https://github.com/near/rainbow-bridge-lib respectively.
```
